src/components/molecules/Preview/Preview.module.scss

```diff
@@ -0,0 +1,21 @@
+.preview {
+  background-color: #ffffff;
+  border: 1px solid #d8dde0;
+
+  :hover {
+    background-color: #ffffff;
+  }
+
+  & tr {
+    border-bottom: 1px solid #d8dde0;
+
+    & td {
+      padding: 1rem;
+    }
+  }
+}
+
+.preview__heading {
+  color: #4c6272;
```

Comment:
TODO: we should probably have a colours.scss file we can put all the custom colours into.

Reply:
I decided to go with colours in the file itself. Any suggestions would be good.